### DISCUSSION TITLE
如何设置查询列表 查询项为必填

### DISCUSSION THREAD

**Discussion by ZhengHuang-20** [ant-design/ant-design#28907] [General]
![image](https://user-images.githubusercontent.com/61463534/104881868-12952a80-5930-11eb-87e8-13b9b06f3db1.png)这个type项在columns中哪个地方设置

**Comment by yoyo837** [comment]
没有上下文背景，不知道你的问题是什么意思，你想做什么事？

**Reply by ZhengHuang-20** [reply]
ProTable 的type 源码里写的是table，这个时候设置columns里设置 
formItemProps: {
      rules: [
        {
          required: true,
          message: '此项为必填项',
        },
      ],
    }是不生效的啊 我截取的图片就是源码这块的逻辑。

但是如果我把ProTable 的type改为form 样式又不是我想要的。

所以我希望能够在ProTable为table 的时候 formItemProps仍然生效

**Comment by yoyo837** [comment]
```js
rules: [{
  required: true,
  message: '请填写xxx',
}]
```

**Reply by ZhengHuang-20** [reply]
我觉的这块完全没有必要判断是table还是form直接改为以下 你看可以不
![image](https://user-images.githubusercontent.com/61463534/104883775-46258400-5933-11eb-8497-eeacd4e84ccc.png)

**Comment by yoyo837** [comment]
ProTable 的可以在那边咨询一下。

**Reply by ZhengHuang-20** [reply]
antd 没有钉钉群？

**Reply by yoyo837** [reply]
这块我个人没有留意。
